quarkus/admin/src/test/java/org/apache/polaris/admintool/el/EclipselinkProfile.java

```diff
@@ -37,6 +37,6 @@ public List<TestResourceEntry> testResources() {
     return List.of(
         new TestResourceEntry(
             PostgresTestResourceLifecycleManager.class,
-            Map.of(INIT_SCRIPT, "org/apache/polaris/admintool/init.sql")));
+            Map.of(INIT_SCRIPT, "org/apache/polaris/admintool/el/init.sql")));
```

Comment:
I'd prefer to defer this change to the PR that actually adds integration tests to the admin tool for the new JDBC persistence impl.